guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -26,18 +26,23 @@ When you set up the compute resource and images for VMware provisioning in {Proj
 * VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
 * The virtual machine builds, `cloud-init` is invoked and calls back {Project} on port `80`, which then redirects to `443`
 
-.Port and Firewall Requirements
+.Prerequisites
+* Configure port and firewall settings to open any necessary connections.
 Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}.
-Ensure that you configure port and firewall settings to open any necessary connections.
-
 ifndef::orcharhino[]
-For more information about port and firewall requirements, see {InstallingServerDocURL}satellite-ports-and-firewalls-requirements_{project-context}[Port and Firewall Requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and Firewalls Requirements] in _Installing {SmartProxyServer}_.
+For more information about port and firewall requirements, see {InstallingServerDocURL}satellite-ports-and-firewalls-requirements_{project-context}[Port and Firewall Requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and Firewalls Requirements] in _{InstallingSmartProxyDocTitle}_.
 endif::[]
+include::snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc[]
 
-.Associating the `userdata` and `Cloud-init` Templates with the Operating System
-. In the {ProjectWebUI}, navigate to *Hosts* > *Operating Systems*, and select the operating system that you want to use for provisioning.
-. Click the *Template* tab.
-. From the *Cloud-init template* list, select *Cloudinit default*.
+.Associating the `Userdata` and `Cloud-init` Templates with the Operating System
+. In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Provisioning Templates*.
+. Search for the *CloudInit default* template and click its name.
+. Click the *Association* tab.
+. Select all operating systems to which the template is applicable and click *Submit*.
+. Repeat the steps above for the *UserData open-vm-tools* template.
+. Navigate to *Hosts* > *Provisioning Setup* > *Operating Systems* and select the operating system that you want to use for provisioning.
```

Comment:
above, you've split "go to X > Y > Z" and "select X". I am OK with either; I lean towards two list items.